deploy/scripts/salt-master-setup.sh

```diff
@@ -326,7 +326,7 @@ install_salt_formula_pkg()
           [ ! -d /srv/salt/reclass/classes/service ] && mkdir -p /srv/salt/reclass/classes/service
 
           declare -a formula_services=("linux" "reclass" "salt" "openssh" "ntp" "git" "nginx" "collectd" "sensu" "heka" "sphinx" "mysql" "grafana" "libvirt" "rsyslog" "glusterfs" "postfix")
-          for formula_service in "${formula_services[@]}"; do
+	  for formula_service in "${formula_services[@]}" $EXTRA_FORMULAS; do
```

Comment:
why the EXTRA_FORMULAS is needed. 
btw I intend to use here the functions, that are capable to process the dependencies as well:https://github.com/salt-formulas/salt-formulas/blob/master/deploy/scripts/formula-fetch.sh#L32